### PR TITLE
Define constant blocks in kube-apiserver

### PR DESCRIPTION
**Pull request by xichengliudui**
/kind cleanup
/priority backlog

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kube-apiserver/app/server.go [hunk 1]
```diff
@@ -84,8 +84,10 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/auth/authenticator/token/bootstrap"
 )
 
-const etcdRetryLimit = 60
-const etcdRetryInterval = 1 * time.Second
+const (
+	etcdRetryLimit    = 60
```
Comment (yue9944882) [inline cmd/kube-apiserver/app/server.go]: what's the point?

Comment (xichengliudui) [reply cmd/kube-apiserver/app/server.go]: Instead of using `const` every time we define a constant, we can define it directly in the constant block.

Comment (xichengliudui) [reply cmd/kube-apiserver/app/server.go]: /cc @yue9944882

**Pull request closed without merging**